firmware/addons/pattern-events.yaml

```diff
@@ -2,32 +2,52 @@
 
 event:
   - platform: template
-    id: doorbell_pattern
-    name: "Doorbell Pattern"
+    id: doorbell_entrance_pattern
+    name: "Entrance Doorbell"
```

Comment:
Why does it cause issues with the binary sensor? Seems to work fine for me since binary sensor is type `binary_sensor` and event is type of `event` in Home Assistant.

Reply:
Oh yea u're right.

But still one thing:
The phone pickup pattern is still called phone pickup pattern, the doorbell patterns however are just like "apartment doorbell" without "pattern".

So would you rename the phone pickup pattern to just phone pickup as well or do you think it's better to keep it with pattern?

And the docs entity names (not id) are not matching the ones in yaml and should follow the same natural naming: Apartment Doorbell (Pattern) / Second Entrance Doorbell (Pattern)

Decide whether to include pattern in the name or not.